packages/calcite-components/src/components/card/card.scss

```diff
@@ -1,46 +1,97 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-card-background-color: Specifies the background color of the component.
+ * @prop --calcite-card-border-color: Specifies the border color of the component.
+ * @prop --calcite-card-box-shadow: Specifies the box shadow of the component.
+ * @prop --calcite-card-corner-radius: Specifies the corner radius of the component.
+ * @prop --calcite-card-accent-color-selected: Specifies the accent color of the component when `selected`.
+ * @prop --calcite-card-selection-background-color: Specifies the background color of the component's selection element.
+ * @prop --calcite-card-selection-background-color-hover: Specifies the background color of the component's selection element when hovered.
+ * @prop --calcite-card-selection-background-color-active: Specifies the background color of the component's selection element when active.
+ * @prop --calcite-card-selection-background-color-selected: Specifies the icon color of the component's selection element when `selected`.
+ * @prop --calcite-card-selection-icon-color: Specifies the icon color of the component's selection element.
+ * @prop --calcite-card-selection-icon-color-hover: Specifies the icon color of the component's selection element when hovered.
+ * @prop --calcite-card-selection-icon-color-selected: Specifies the icon color of the component's selection element when `selected`.
+ *
+ */
+
 :host {
   @apply block max-w-full;
-  & .calcite-card-container {
-    @apply bg-foreground-1
-    border-color-2
-    text-color-3
-    relative
+  --calcite-card-corner-radius: var(--calcite-corner-radius-sharp);
+  --calcite-card-background-color: var(--calcite-color-foreground-1);
+  --calcite-card-border-color: var(--calcite-color-border-3);
+  --calcite-card-box-shadow: var(--calcite-shadow-none);
+  --calcite-card-accent-color-selected: var(--calcite-color-brand);
```

Comment:
Sidebar: We should make sure we use the same naming for accent props. I recently added one using `highlight`. I much prefer accent. 🏆 cc @alisonailea